region/maridia/inner-yellow.json

```diff
@@ -416,6 +416,48 @@
                     "This is applicable using the Power Bomb block platform in PCJR or the normal platform in Tourian Escape Room 2."
                   ]
                 },
+                {
+                  "name": "Cross Room Jump - Spring Ball Jump Above",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "minHeight": 7,
+                      "maxHeight": 7,
+                      "maxLeftPosition": 1,
+                      "minRightPosition": 1
+                    }},
+                    "canCrouchJump",
```

Comment:
This doesn't need a crouch.
So you can do it from PCJR powerbomb blocks if you break spin too.

Reply:
`canSpringBallBombJump` works at 7 but not 8, so seems pointless.

Reply:
Oh I think I understand, you weren't saying this strat here is pointless, just a hypothetical one using Bombs